zephyr/ra/ra_cfg/fsp_cfg/bsp/ra6m3/bsp_mcu_family_cfg.h

```diff
@@ -6,8 +6,8 @@
 
 #ifndef BSP_MCU_FAMILY_CFG_H_
 #define BSP_MCU_FAMILY_CFG_H_
+#include "bsp_mcu_device_pn_cfg.h"
```

Comment:
Why is this change needed ?

Reply:
> Why is this change needed ?

This file includes `bsp_mcu_info.h` twice and is missing the `bsp_mcu_device_pn_cfg.h` file, which defines the part number and Flash size macro for RA6M3.
Currently, these boards lack information about Code flash size and Data flash size. So I have added it into file `bsp_mcu_device_pn_cfg.h` and if it isn't included in the main file config (`bsp_mcu_family_cfg.h`), the flash size information can not be used.